src/utils/components/AlertsCard/utils/utils.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { TFunction } from 'i18next';
+
+import {
+  ALL_ALERTS,
+  VIRTUALIZATION_ONLY_ALERTS,
+} from '@kubevirt-utils/components/AlertsCard/utils/constants';
+import { AlertType, SimplifiedAlerts } from '@kubevirt-utils/components/AlertsCard/utils/types';
+import {
+  PrometheusLabels,
+  YellowExclamationTriangleIcon,
+} from '@openshift-console/dynamic-plugin-sdk';
+import {
+  BlueInfoCircleIcon,
+  RedExclamationCircleIcon,
+} from '@openshift-console/dynamic-plugin-sdk/lib/app/components/status/icons';
+import ExclamationCircleIcon from '@patternfly/react-icons/dist/js/icons/exclamation-circle-icon';
+import ExclamationTriangleIcon from '@patternfly/react-icons/dist/js/icons/exclamation-triangle-icon';
+
+export const color = {
+  [AlertType.critical]: 'red',
+  [AlertType.warning]: 'orange',
+  [AlertType.info]: 'blue',
+};
+
+export const icon = {
+  [AlertType.critical]: <ExclamationCircleIcon />,
+  [AlertType.warning]: <ExclamationTriangleIcon />,
+  [AlertType.info]: <ExclamationCircleIcon />,
+};
+
+export const filterAlerts = (sortedAlerts: SimplifiedAlerts) => {
+  const filteredAlerts = { critical: [], warning: [], info: [] };
+
+  for (const [key, value] of Object.entries(sortedAlerts)) {
+    filteredAlerts[key] = value.filter((alert) => !alert?.isVMAlert);
+  }
+
+  return filteredAlerts;
+};
+
+export const getAlertIcon = (alertType: AlertType) => {
+  switch (alertType) {
+    case AlertType.critical:
+      return () => <RedExclamationCircleIcon title="Critical" />;
+    case AlertType.warning:
+      return () => <YellowExclamationTriangleIcon title="Warning" />;
+    case AlertType.info:
+      return () => <BlueInfoCircleIcon title="Information" />;
+    default:
+      return null;
+  }
+};
+
+export const createAlertKey = (activeAt: string, labels: PrometheusLabels) =>
+  [activeAt, labels?.name, labels?.vmName, labels?.pod, labels?.uid, labels?.instance]
```

Comment:
labels?.uid is enough?

Reply:
Not all labels have a `uid` label. I added multiple labels to help ensure the key is unique as not all alerts have all of the same labels. It would much easier if all alerts had a `uid` label, though.